python/test/unit/tools/test_aot.py

```diff
@@ -63,6 +63,24 @@ def kernel(C, A, B, M, N, K,
   tl.store(c_ptrs, c)
 """
 
+gluon_kernel_src = """
+from triton.experimental import gluon
+from triton.experimental.gluon import language as gl
+
+@gluon.jit
+def kernel(
+    C, A, B, M, N, K,
+    stride_cm, stride_cn,
+    stride_am, stride_ak,
+    stride_bk, stride_bn,
+    BLOCK_M: gl.constexpr,
+    BLOCK_N: gl.constexpr,
+    BLOCK_K: gl.constexpr
+):
+    a = gl.load(A)
```

Comment:
This is Triton effectively without any explicit layout. You can add some layouts to make it really Gluon, e.g., like [this kernel with one layout](https://github.com/triton-lang/triton/blob/0a4cbff20edc4079c1a87624d90697906cab515e/python/test/gluon/test_core.py#L46).